ansible/roles/ocp4-workload-sso-cloud-native-apps/files/codeready_cr.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: org.eclipse.che/v1
```

Comment:
Why not use the workload `ocp4_workload_codeready_workspaces` that's already there. No need to re-invent the wheel.

Reply:
I need more time to this. will take this at a later stage of the lab.